speedy-recovery-client/src/service/BackendMappingExtractionUtils.js

```diff
@@ -38,23 +53,23 @@ const getUserId = (conversationResource, id) => {
   return missingField;
 };
 
-const getTitle = (conversationResource, id, conversationList) => {
+const getTitle = (conversationResource, id, userList) => {
   if (
     conversationResource &&
     conversationResource.userid1 &&
     conversationResource.userid2 &&
-    conversationList
+    userList
   ) {
     if (conversationResource.userid1 === id) {
-      for (let conversation of conversationList) {
-        if (conversation.id === conversationResource.userid2) {
-          return conversation.name;
+      for (let user of userList) {

```

Comment:
A nicer way of doing this might be to do something like 
`userList.find(user => {
if (user.id === conversationResource.userid2) {
          return user.name;
        }`
      }